src/common/form-group.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react'
+import { Col, Row } from 'grid'
+import { propTypes } from 'utils'
+
+const FormGroup = propTypes({
+  label: propTypes.any.isRequired,
+  children: propTypes.any.isRequired
+})(({ label, children }) => (
+  <Row className='form-group'>
+    <label className='col-sm-2 form-control-label'>{label}</label>
+    <Col mediumSize={10}>
+      {children}
+    </Col>
+  </Row>
+))
+export { FormGroup as default }
```

Comment:
This module should be named `form-grid` and expose the following components:

- `Row`
- `LabelCol`: to be used inside a `FormGrid.Row` and expect a content as children (text, node, etc.) to be used as the label
- `InputCol` to be used inside a `FormGrid.Row` and expect an input as children